README.md

```diff
@@ -1,54 +1,191 @@
 # BizHawk
 
-BizHawk is a multi-system emulator written in C#. BizHawk provides nice features for casual gamers such as fullscreen, rewind, and joypad support in addition to rerecording and debugging tools for all system cores.
+[![GitHub latest release](https://img.shields.io/github/release/TASVideos/BizHawk.svg?logo=github&logoColor=333333&style=popout)](https://github.com/TASVideos/BizHawk/releases/latest)
+[![dev builds | AppVeyor](https://img.shields.io/badge/dev_builds-AppVeyor-orange.svg?logo=appveyor&logoColor=333333&style=popout)](https://ci.appveyor.com/project/zeromus/bizhawk-udexo/history)
+[![unique systems emulated | 22](https://img.shields.io/badge/unique_systems_emulated-22-darkgreen.svg?logo=buffer&logoColor=333333&style=popout)](https://github.com/TASVideos/BizHawk/blob/master/README.md#features)
 
-[Release Notes](http://tasvideos.org/Bizhawk/ReleaseHistory.html)
+A multi-system emulator written in C#. As well as quality-of-life features for casual players, it also has recording/playback and debugging tools, making it the first choice for TASers (Tool-Assisted Speedrunners).
 
-[Here](http://tasvideos.org/Bizhawk/Features.html) is a list of features offered by bizhawk.
+tl;dr: click the "latest" button above to grab it. The changelog is [here at TASVideos](http://tasvideos.org/Bizhawk/ReleaseHistory.html). **Don't mix versions**, copy `config.ini` to keep settings.
```

Comment:
- Remove `rl;dr:`.
- There's no "latest" button above.
- The warning is "Never mix different versions of BizHawk. Keep each version in its own folder."
- Sharing the config file is discouraged, it can screw with new, old, or changed features. If one does it, no guarantees are made.

Reply:
you can't copy config.ini? guess I've been lucky. fixed